TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/kafka/KafkaBehavioralPolicy.java

```diff
@@ -9,7 +9,7 @@
 public class KafkaBehavioralPolicy {
 
     // Default behavior to log and ignore invalid records
```

Comment:
Can you supply some more detailed docs now that the behavior has changed?  What should a user expect if an Exception is returned?

Reply:
Added a proper javadoc